css-view-transitions-1/Overview.bs

```diff
@@ -682,21 +682,11 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 	</div>
 
 	<div algorithm="suppress rendering">
-		1. For each {{Document}} in <var ignore>docs</var> with a [=document/transition suppressing rendering=] of true:
-
-			Issue: Move this to the html spec.
-				See [html/8368](https://github.com/whatwg/html/pull/8368/files).
-
-			Issue: How should input be handled when in this state?
-				The last frame presented to the user will not reflect the DOM state as it asynchronously switches to the new version.
-				See [issue 7797](https://github.com/w3c/csswg-drafts/issues/7797).
-
-			Note: The aim is to prevent unintended DOM updates from being presented to the user after a cached snapshot for the elements has been captured.
-				We wait for one [=rendering opportunity=] after prepare to present DOM mutations made by the author before prepare to be presented to the user.
-				This is also the content captured in snapshots.
+		A navigable has no rendering opportunities</span> if its active document is render-blocked;
```

Comment:
I think there's been an editing error here? Seems incomplete.

Reply:
We need one line to be added to [this](https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering:~:text=A%20navigable%20has%20no,occur%20at%20regular%20intervals.) paragraph:

```
A [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) has no [rendering opportunities](https://html.spec.whatwg.org/multipage/webappapis.html#rendering-opportunity) if its [active document](https://html.spec.whatwg.org/multipage/document-sequences.html#nav-document) is [render-blocked](https://html.spec.whatwg.org/multipage/dom.html#render-blocked); otherwise, [rendering opportunities](https://html.spec.whatwg.org/multipage/webappapis.html#rendering-opportunity) are determined based on hardware constraints such as display refresh rates and other factors such as page performance or whether the document's [visibility state](https://html.spec.whatwg.org/multipage/interaction.html#visibility-state) is "visible". Rendering opportunities typically occur at regular intervals.
```

changes to:

```
A [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) has no [rendering opportunities](https://html.spec.whatwg.org/multipage/webappapis.html#rendering-opportunity) if its [active document](https://html.spec.whatwg.org/multipage/document-sequences.html#nav-document) is [render-blocked](https://html.spec.whatwg.org/multipage/dom.html#render-blocked);  or its <span>active document</span>'s has <span>transition suppressing rendering</span> set to true; otherwise, [rendering opportunities](https://html.spec.whatwg.org/multipage/webappapis.html#rendering-opportunity) are determined based on hardware constraints such as display refresh rates and other factors such as page performance or whether the document's [visibility state](https://html.spec.whatwg.org/multipage/interaction.html#visibility-state) is "visible". Rendering opportunities typically occur at regular intervals.
```

Should I copy over the complete paragraph with the above addition from the html spec?

Reply:
Right now the edit has a closing span without an opening span, which is why I thought there was a copy/paste error.

I'd go for something like this:

---

In the definition for [=rendering opportunity=], add the following condition:

A [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable) has no [rendering opportunities](https://html.spec.whatwg.org/multipage/webappapis.html#rendering-opportunity) if active document's has transition suppressing rendering set to true.

Reply:
Done.